flexmeasures/api/v3_0/__init__.py

```diff
@@ -11,6 +12,21 @@ def register_at(app: Flask):
 
     v3_0_api_prefix = "/api/v3_0"
 
+    def cost_function() -> int:
+        if request.endpoint == "SensorAPI:trigger_schedule":
+            return 1
+        return 0
+
+    # Apply rate limit: a schedule can be triggered once per 5 minutes per sensor per account
+    SensorAPI.decorators.append(
+        app.limiter.limit(
+            "1 per 5 minutes",
```

Comment:
We apply this here for all endpoints, but the timing we want to allow should differ per endpoint.

Reply:
But given the cost function, it is only effectively applied to one endpoint. We are free to add more `limiter.limit()` decorators with other rate limits that affect other endpoints (via the cost function).